typescript/api-client/projects/vcd/schematics/src/schematics/plugin-seed/templates/wellington-xendi/__name@dasherize__/src/public/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "urn": "vmware:vcloud:plugin:<%= dasherize(vendor) %>:<%= dasherize(name) %>",
+    "name": "<%= classify(name) %>",
+    "containerVersion": "9.1.0",
+    "version": "1.0.0",
+    "scope": ["service-provider", "tenant"],
+    "permissions": [],
+    "description": "An example plugin.",
+    "vendor": "<%= dasherize(vendor) %>",
+    "license": "Copyright (C) VMware 2017-2018.  All rights reserved.",
```

Comment:
let's make this BSD-2-Clause...no need for everyone to get a VMware Copyright by default.